manager/state/store/services.go

```diff
@@ -26,6 +27,11 @@ func init() {
 					Unique:  true,
 					Indexer: serviceIndexerByName{},
 				},
+				indexRuntime: {
+					Name:         indexRuntime,
+					AllowMissing: true,
+					Indexer:      serviceIndexerByRuntime{},
+				},
```

Comment:
Discussed IRL - this is to allow filtering by container for things such as `docker service ls`